src/rules/rdf/mustNotExist.js

```diff
@@ -4,7 +4,7 @@ import * as messages from 'messages';
 
 
 export function mustNotExist(xmlDoc, filename=null,
-                             namespace='http://www.mozilla.org/2004/em-rdf#') {
+    {namespace='http://www.mozilla.org/2004/em-rdf#'}={}) {
```

Comment:
Doesn't this get passed by default in the options? If we do this can be {namespace}={}